c2rust-analyze/src/dataflow/type_check.rs

```diff
@@ -109,7 +110,26 @@ impl<'tcx> TypeChecker<'tcx, '_> {
 
         match *rv {
             Rvalue::Use(ref op) => self.visit_operand(op),
-            Rvalue::Repeat(..) => todo!("visit_rvalue Repeat"),
+            Rvalue::Repeat(ref op, _) => {
+                assert!(lty.ty.is_array());
```

Comment:
This is always the case, right?  So this is just a sanity check?

Reply:
Yes it's just making the same check that's made in the `Aggregate`/`Array` case